views/js/qtiCreator/editor/ckEditor/htmlEditor.js

```diff
@@ -613,13 +613,12 @@ define([
          *
          * @param {JQuery} $editable
          */
-        getData : function($editable){
-            var editor = $editable.data('editor');
-            if(editor){
+        getData : function($editable) {
+            const editor = $editable.data('editor');
+            if (editor) {
                 return _htmlEncode(editor.getData());
-            }else{
-                throw new Error('no editor attached to the DOM element');
             }
+            return false
```

Comment:
**nitpick:** Not sure this is the best approach. The return value will be either a boolean or a string... Why not keeping the exception and properly handle it on the consumer?

Reply:
new solution will be provided